### ISSUE TITLE
Unable to Access HOOBS via the Web Interface

### ISSUE THREAD

**Issue by PBXLthrMan** [hoobs-org/HOOBS#1569]
The Hoobs Box is accessible and is able to be controlled thru the MAC app but is unable to load the dashboard thru a web page.  

I have cleared the cache on multiple occasions and even still doesn't load the dashboard.

This is an issue no matter what browser I use (Chrome/Safari, Firefox).

I spent some significant time troubleshooting this issue with Eddie D. from HOOBS, he was not sure what could be the issue and advised me to open up a GitHub issue.

**Comment by mkellsy** [comment]
This is why the app exists. Some routers don't allow mDNS addresses.
If http://hoobs.local doesn't work try http://hoobs or http://192.168.X.X (replace 192.168.X.X with the actual IP address)

The app uses the IP address.

**Comment by mkellsy** [comment]
And it works with the app?

**Comment by PBXLthrMan** [comment]
> OK in the app go to Menu -> Terminal Username is **hoobs** passwd is **hoobsadmin**
> 
> Then type this command
> 
> ```
> ip address
> ```
> 
> Post the results. (You can copy the text by selecting it and then right clicking)

hoobs@hoobs:~ $ 
hoobs@hoobs:~ $ ip address
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether b8:27:eb:6f:6c:bf brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.64/24 brd 192.168.1.255 scope global dynamic noprefixroute eth0
       valid_lft 81353sec preferred_lft 81353sec
3: wlan0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc pfifo_fast state DOWN group default qlen 1000
    link/ether 2e:50:ff:55:27:e3 brd ff:ff:ff:ff:ff:ff
hoobs@hoobs:~ $

**Comment by mkellsy** [comment]
OK the web interface should be here
http://192.168.1.64

**Comment by PBXLthrMan** [comment]
> OK the web interface should be here http://192.168.1.64

Yup, and not loading. I get the icon on the tab, but nothing loading.

**Comment by PBXLthrMan** [comment]
> What browser are you using?

Any, Chrome, Safari, Firefox.

**Comment by mkellsy** [comment]
What browser are you using right now?

**Comment by PBXLthrMan** [comment]
> 1. You have an invalid extension in Chrome. Go to your extensions and see what it tells you.
> 2. Try clearing your browser cache. Chrome Menu -> History -> History. Then click on "Clear browser data"
>    Then set the "Time range" to "All time"
>    Then only select "Cached images and files"
>    Then click "Clear data".
> 
> After this refresh and the UI should show.

I removed all of the extension, cleared the cache, rebooted (for good measure), still the same results whether using the IP address or the DNS entry. Nothing pops up.

**Comment by mkellsy** [comment]
The other thing you can try is to re-install the GUI.

From the terminal run this command
```
sudo apt-get --reinstall install hoobs-gui
```

Then restart the service
```
sudo hoobsd service restart
```

**Comment by PBXLthrMan** [comment]
> The other thing you can try is to re-install the GUI.
> 
> From the terminal run this command
> 
> ```
> sudo apt-get --reinstall install hoobs-gui
> ```
> 
> Then restart the service
> 
> ```
> sudo hoobsd service restart
> ```

That did the trick! Thank you so much. Fhew that was painful.
